dsfs/dataset.go

```diff
@@ -42,16 +41,16 @@ func LoadDataset(store cafs.Filestore, path string) (*dataset.Dataset, error) {
 func LoadDatasetRefs(store cafs.Filestore, path string) (*dataset.Dataset, error) {
 	ds := dataset.NewDatasetRef(path)
 
-	path = PackageFilepath(store, path, PackageFileDataset)
-	data, err := fileBytes(store.Get(path))
+	pathWithExt := PackageFilepath(store, path, PackageFileDataset)
```

Comment:
When I hear "extension" I think of ".json", not "dataset.json". Would `pathWithBasename` be a better term?

Reply:
in2it